package.json

```diff
@@ -22,5 +22,6 @@
     "prettier": "^2.8.4",
     "tailwindcss": "^3.2.7",
     "typedoc": "^0.23.25"
-  }
+  },
+  "packageManager": "yarn@3.5.0"
```

Comment:
Does this mean we are using corepack? Does using corepack mean we could get rid of the .yarn/releases/yarn-3.5.0.cjs that we check in?

Reply:
Oh, you are right! I assumed it would not be possible since none of our other repos do that. But I just tested it and seems to work great.